docs/usage.md

```diff
@@ -38,7 +38,7 @@ _deptry_ can also be configured to look for `requirements.txt` files with other
 
 To determine issues with imported modules and dependencies, _deptry_ will scan the working directory and its subdirectories recursively for `.py` and `.ipynb` files, so it can
 extract the imported modules from those files. Any files solely used for development purposes, such as files used for unit testing, should not be scanned. By default, the directories
-`venv`, `.venv`, `tests`,`.git` and the file `setup.py` are excluded.
+`venv`, `.venv`, `.direnv`, `tests`,`.git` and the file `setup.py` are excluded.
```

Comment:
```suggestion
`venv`, `.venv`, `.direnv`, `tests`, `.git` and the file `setup.py` are excluded.
```

Reply:
Thanks just fixed it because of the merge conflict :)